src/main/java/edu/cornell/kfs/module/purap/document/service/impl/CuB2BPurchaseOrderSciquestServiceImpl.java

```diff
@@ -117,12 +116,15 @@ public String sendPurchaseOrder(PurchaseOrderDocument purchaseOrder) {
             LOG.debug("sendPurchaseOrder() Generating cxml");
             String cxml = getCxml(purchaseOrder, requisitionInitiatorPrincipalId, getB2bPurchaseOrderPassword(), contractManager, contractManagerEmail, vendorDuns, true);
 
-            LOG.debug("sendPurchaseOrder() Sending cxml\n" + cxml);
-            LOG.debug("sendPurchaseOrder() Sending cxml\n" + cxml);
+            LOG.info("sendPurchaseOrder() Sending cxml\n{}", cxml);
             String responseCxml = b2bDao.sendPunchOutRequest(cxml, getB2bPurchaseOrderURL());
             LOG.info("b2bPurchaseOrderURL " + getB2bPurchaseOrderURL());
 
-            LOG.info("sendPurchaseOrder(): Response cXML for po #" + purchaseOrder.getPurapDocumentIdentifier() + ":\n" + responseCxml);
+            LOG.info(
+                    "sendPurchaseOrder(): Response cXML for po #{}:\n{}",
+                    purchaseOrder.getPurapDocumentIdentifier(),
+                    responseCxml

```

Comment:
That results in an error I am not sure how to fix.

Reply:
Here's how that statement is written in the 02/23/2022 version of `B2BPurchaseOrderSciquestServiceImpl`. Does this work or does it still give you an error?

```java
            LOG.info(
                    "sendPurchaseOrder(): Response cXML for po #{}:\n{}",
                    purchaseOrder::getPurapDocumentIdentifier,
                    () -> responseCxml
            );
```

If you'd rather deal with this in a follow-up ticket instead, please let me know.

Reply:
Hi Chad, I have tried the code from the base class but I am getting this error: "Local variable responseCxml defined in an enclosing scope must be final or effectively final". I cannot make responseCxml final as it is further changed down the code. Does the code compile successfully for you when you make that change? Thank you

Reply:
It looks like our code is modifying the `responseCxml` variable in multiple spots, which would explain why you're getting that error. One possible fix would be to declare another final (or effectively final) variable just prior to the log statement that has the value of the `responseCxml` one, then modify the log statement to use the new variable instead.

If you would prefer to handle this in a separate ticket, that's fine with me.

Reply:
Sounds good. Thanks!

Reply:
Since I had some time left at the end of the day I made the changes that Chad suggested on this PR and committed them. This way we may not need to adjust the code on a follow up user story.